Code/Editor/Platform/Windows/editor_lib_windows.cmake

```diff
@@ -5,3 +5,8 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 #
 #
+
+set(LY_BUILD_DEPENDENCIES
+    PRIVATE
+        Gdi32.lib
+)
```

Comment:
We have found out in [discord](https://discord.com/channels/805939474655346758/816043694762754088/880878761359310888) that nothing in the Editor is using Gdi32.lib

Reply:
removing